fdbserver/networktest.actor.cpp

```diff
@@ -33,21 +39,38 @@ NetworkTestInterface::NetworkTestInterface( INetwork* local )
 	test.makeWellKnownEndpoint( WLTOKEN_NETWORKTEST, TaskPriority::DefaultEndpoint );
 }
 
+constexpr double startingDelay = 5.0;
+constexpr int packetSize = 155 * 1024;	// slightly larger than 150K limit
+constexpr int NUM_PARALLEL_CLIENT = 32;
+
+void generateReplyString(std::string& output, const int length) {
+	std::ifstream ifs("/dev/random");
```

Comment:
Don't use /dev/random.  It's extremely slow and breaks determinism.   Use deterministicRandom() and / or std::mt19937 instead (see flow/IndexedSet.cpp for examples).